src/current/v24.3/cockroach-encode-uri.md

```diff
@@ -0,0 +1,66 @@
+---
+title: cockroach encode-uri
+summary: Construct a connection URI that a client or application can use to connect to a cluster, based on the command-line arguments you provide.
+toc: true
+docs_area: reference.cli
+---
+
+The `cockroach encode-uri` command constructs a connection URI that a client or application can use to connect to a cluster, based on the command-line arguments you provide. The returned connection URI is in `postgresql:/` format.
```

Comment:
```suggestion
The `cockroach encode-uri` command constructs a connection URI that a client or application can use to connect to a cluster, based on the command-line arguments you provide. The returned connection URI is in `postgresql://` format.
```

We only ever want to refer to the scheme as either `postgresql` or `postgresql://`. `postgres:/` will not do what the user thinks.